controllers/insights/index.js

```diff
@@ -71,7 +71,11 @@ router.get(
                 pagination: research.pagination,
             });
         } catch (error) {
-            next(error);
+            if (error.statusCode >= 500) {
```

Comment:
I _think_ this might need to change, started taking a look at this on Thursday and noticed that this response format changed slightly with `got`. This might need to be something like `error.response.statusCode`

Reply:
Hmm, it worked locally when I tried it...?